dask/dataframe/io/parquet.py

```diff
@@ -121,7 +117,8 @@ def read_parquet(path, columns=None, filters=None, categories=None, index=None,
                         columns=[c for c in pf.columns if c in dtypes])
 
     for cat in categories:
```

Comment:
(to answer in full: the dtype of the values are always know beforehand in the global schema, i.e., pure metadata, only the specific encoding of how that is stored - dictionary or not - is undefined at the global level)